ci.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Fixed `minirust-tooling` commit, we need to bump this occasionally.
+TOOLING_COMMIT="0441a98"
+
+git clone "https://github.com/memoryleak47/minirust-tooling" ~/minirust-tooling
+ln -s ~/work/minirust/minirust ~/minirust-tooling/minirust
+
+cd ~/minirust-tooling
+git checkout "$TOOLING_COMMIT"
+
+(cd specr-transpile; cargo run)
```

Comment:
This cannot be `cargo run --manifest-path specr-transpile`, as `specr-transpile` needs this working directory.
That's also something I intend to improve later on.